pkg/config/tlscfg/ciphersuites.go

```diff
@@ -40,3 +48,19 @@ func CipherSuiteNamesToIDs(cipherNames []string) ([]uint16, error) {
 	}
 	return ciphersIDs, nil
 }
+
+// VersionNameToID returns the version ID from version name
+func VersionNameToID(versionName string) (uint16, error) {
+	if len(versionName) == 0 {
+		return DefaultVersion(), nil
+	}
+	if version, ok := versions[versionName]; ok {
+		return version, nil
+	}
+	return 0, fmt.Errorf("unknown tls version %q", versionName)
+}
+
+func DefaultVersion() uint16 {
+	// TLS1.2 is considered the safest default
```

Comment:
tls.Config says default is 1.0. I don't think we need to invent our own default - if the user did not set the value then we don't pass anything to tls.Config